frontend/src/hooks/useGetTopLevelOperations.ts

```diff
@@ -10,7 +10,7 @@ type UseGetTopLevelOperations = (
 const useGetTopLevelOperations: UseGetTopLevelOperations = (queryKey) =>
 	useQuery<ServiceDataProps>({
 		queryKey,
-		queryFn: getTopLevelOperations,
+		queryFn: (): Promise<ServiceDataProps> => getTopLevelOperations({}),
```

Comment:
couldn't find any reference of this code being used, hence not passing the params here.

Reply:
This one https://github.com/SigNoz/signoz/blob/1281330c52877ca893f3f87b6e7cf488875e0f98/frontend/src/container/ServiceApplication/ServiceMetrics/index.tsx#L5-L35

Reply:
follow the caller chain and you ll end up in a component which is not being utilised anywhere

Reply:
If you look at the commit that introduced the change, it can be noticed that it is built to support a feature flag. It might not be used but it will be used. Ideally, `ServicesUsingMetrics` should have been used somewhere with the feature flag `USE_SPAN_METRIC`.

Reply:
let me check this again and see what was the idea behind the same